src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -320,12 +320,13 @@ private static IDictionary<string, string> GetWorkerCapabilities(WorkerOptions w
             }
 
             // Add required capabilities; these cannot be modified and will override anything from WorkerOptions
-            capabilities["RpcHttpBodyOnly"] = bool.TrueString;
-            capabilities["RawHttpBodyBytes"] = bool.TrueString;
-            capabilities["RpcHttpTriggerMetadataRemoved"] = bool.TrueString;
-            capabilities["UseNullableValueDictionaryForHttp"] = bool.TrueString;
-            capabilities["TypedDataCollection"] = bool.TrueString;
-            capabilities["WorkerStatus"] = bool.TrueString;
+            capabilities[WorkerCapabilities.RpcHttpBodyOnly] = bool.TrueString;
+            capabilities[WorkerCapabilities.RawHttpBodyBytes] = bool.TrueString;
+            capabilities[WorkerCapabilities.RpcHttpTriggerMetadataRemoved] = bool.TrueString;
+            capabilities[WorkerCapabilities.UseNullableValueDictionaryForHttp] = bool.TrueString;
+            capabilities[WorkerCapabilities.TypedDataCollection] = bool.TrueString;
+            capabilities[WorkerCapabilities.WorkerStatus] = bool.TrueString;
+            capabilities[WorkerCapabilities.IncludeEmptyEntriesInMessagePayload] = bool.TrueString;
```

Comment:
Instead of sending this always as enabled, should we just set the default value to "enabled". This allows users to use the previous behavior if they choose to do so by overriding the worker option property in their bootstrapping code.

Reply:
I must have misunderstood the task, I thought we were trying to get rid of the capability as an option. I can bring it back and set it as default behaviour instead!

Reply:
I am assuming it's similar for the enable user code change?